README.md

```diff
@@ -54,6 +54,12 @@ like `pipx install --editable .`
 If you do not want this behaviour, use: `uv tool install --from . xcp-ng-dev`
 or `pipx install .`
 
+## Completion
+
+### Fish
+
+To install the completion, run `register-python-argcomplete --shell fish xcp-ng-dev > ~/.config/fish/completions/xcp-ng-dev.fish` and relaunch fish.
+
```

Comment:
Docs for bash and zsh would be nice too. CC @glehmann

Reply:
I never know where to put the zsh completion. Do we have a power zsh user?